### ISSUE TITLE
huber loss doc error

### ISSUE THREAD

**Issue by czr-gc** [PaddlePaddle/Paddle#44019]
### bug描述 Describe the Bug

huber_loss的公式错误：

https://github.com/PaddlePaddle/Paddle/blob/f1ffd59a8686cda32dae65aff259d489fff83da7/python/paddle/fluid/layers/loss.py#L1485

应该是
```
huber_loss = delta * abs(label - input) - 0.5 * delta * delta 
```

### 其他补充信息 Additional Supplementary Information

kernel代码中用的是abs_val：
```
  HOSTDEVICE T operator()(const T& val) const {
    T abs_val = std::abs(val);
    if (abs_val <= delta) {
      return static_cast<T>(0.5) * val * val;
    } else {
      return delta * (abs_val - static_cast<T>(0.5) * delta);
    }
  }
```

**Comment by betterpig** [comment]
感谢你的建议，我们后续会修复该错误。

**Comment by betterpig** [comment]
你好，经了解该API将被废弃，请使用另一API  [paddle.nn.SmoothL1Loss](https://www.paddlepaddle.org.cn/documentation/docs/zh/develop/api/paddle/nn/SmoothL1Loss_cn.html)
